src/components/my-history/custom-my-history.vue

```diff
@@ -16,12 +16,28 @@
             @click="closeModal"
             data-test="my-history-query"
             :suggestion="suggestion"
-            suggestionClass="x-suggestion"
+            suggestionClass="x-suggestion x-w-[320px]"
           >
             <HistoryIcon class="max-desktop:x-icon-lg" />
 
             <div class="x-flex x-flex-col x-gap-2">
-              <p>{{ suggestion.query }}</p>
+              <p class="hover:x-underline">{{ suggestion.query }}</p>
```

Comment:
This way you only get the underline when hovering this specific element. This underline should appear when you hover anywhere in the parent.
Use [tailwind group](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state) for this